test/sources/test3.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: build.knative.dev/v1alpha1
+kind: Build
+metadata:
+  name: test-hybrid-sources
+  labels:
+    expect: succeeded
+spec:
+  sources:
+  - name: basel
+    git:
+      url: https://github.com/bazelbuild/rules_docker      
+      revision: master
+  - name: "gcs-source"
+    gcs:
+      type: Archive
+      location: gs://build-crd-tests/rules_docker-master.zip
```

Comment:
Do these tests run on presubmit?

Reply:
Yes. presubmit runs both e2e go tests and yaml tests